example/src/main/java/org/eclipse/hono/example/ExampleReceiver.java

```diff
@@ -79,7 +79,7 @@ private void onDisconnect(final ProtonConnection con) {
 
         // give Vert.x some time to clean up NetClient
         vertx.setTimer(DEFAULT_CONNECT_TIMEOUT_MILLIS, reconnect -> {
-            LOG.info("attempting to re-connect to Hono server");
+            LOG.info("attempting to re-connect to Hono Messaging");
```

Comment:
Since this is just the ExampleSender/Receiver, I think we should not make it too technical and instead just say `attempting to re-connect to Hono` ...